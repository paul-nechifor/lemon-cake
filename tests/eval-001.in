(eval 1)
(eval 'a')
(eval ())
(eval (dict 1 2))
(eval :(+ 1 2))
(eval (list :+ 1 3))
((eval :(~ (a b) (* a b))) 3 6)
(eval :(= a 3))
(eval :(- a 1))
