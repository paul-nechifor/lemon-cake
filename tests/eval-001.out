1
'a'
()
(dict 1 2)
3
4
18
3
2
